### PR TITLE
Add a new blog 0409

### PR DESCRIPTION
**Pull request by booooodv**

### CODE REVIEW

**Comment by booooodv** [comment]
@husiyu PTAL

File: the-overview-of-tidb-4.0.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+---
+title: The Overview of TiDB 4.0
+author: ['唐刘']
+date: 2020-04-09
```
Comment (husiyu) [inline the-overview-of-tidb-4.0.md:4]: 2020-04-08

File: the-overview-of-tidb-4.0.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+**是不是很酷？更酷的是，TiUP 会管理 TiDB 整个生态里面的组件，无论是 TiDB、TiFlash，还是生态工具，都可以通过 TiUP 来进行管理和使用，用户也可以给 TiUP 添加对应的组件工具。**
+
+## OLTP or OLAP，还是个难题吗？
+
+我的业务到底是 OLTP 还是 OLAP？我们相信，很多时候，用户都不能很清楚的回答出来。但我们知道，用户最想要的是**「无论我的业务是怎样的，我要在你的数据库里面都能快速的跑出来结果」**。
+
+这个需求看似简单，要满足却是非常的困难，但是在 TiDB 4.0，我们终于可以很自豪的说，离彻底搞定这个需求更接近了一步，因为我们提供一套完备的 Hybrid transaction/analytical processing (HTAP) 解决方案，那就是 [TiDB + TiFlash](https://pingcap.com/blog-cn/10x-improving-analytical-processing-ability-of-tidb-with-tiflash/) 。
+
+![1-tidb-tispark](media/the-overview-of-tidb-4.0/1-tidb-tispark.png)
```
Comment (husiyu) [inline the-overview-of-tidb-4.0.md:46]: tidb-tiflash , there is no tispark...

File: the-overview-of-tidb-4.0.md (hunk 1 above)
Comment (booooodv) [reply the-overview-of-tidb-4.0.md:4]: done

File: the-overview-of-tidb-4.0.md (hunk 2 above)
Comment (booooodv) [reply the-overview-of-tidb-4.0.md:46]: done

**Comment by YiniXu9506** [comment]
LGTM

**Pull request merged**